doc/website/getting-started/installation.md

```diff
@@ -4,6 +4,26 @@ iceoryx_utils and iceoryx_posh are deployed as independent cmake packages. Posh
 
 ## Prerequisites
 
+### General
+
+ - [cmake](https://cmake.org), 3.5 or later
+ - One of the following compilers:
+   - [gcc](https://gcc.gnu.org), 7.4 or later 
+   - [clang](https://clang.llvm.org), 9.0 or later
+   - [msvc](https://visualstudio.microsoft.com/de/), part of Visual Studio 2019 or later
+ - optional, [ncurses](https://invisible-island.net/ncurses/), 6.2 or later. Required by introspectiont tool.
```

Comment:
I would apply single source principle here. We already mention parts of this info in the `CONTRIBUTING.md`. As this info is needed by users and not just by contributors, could you please delete the dependencies section in `CONTRIBUTING.md`? Also add `acl`  and `POSIX` here.

Reply:
POSIX is in our context misleading since it refers to the operating system and we support non posix systems like Mac OS and Windows and also do not support every POSIX system for instance FreeBSD.

I will add `acl`.